pkg/nuclide-ocaml-rpc/lib/MerlinService.js

```diff
@@ -89,6 +97,11 @@ export async function errors(
   return instance ? instance.errors() : null;
 }
 
+export async function outline(path: NuclideUri) {
```

Comment:
This needs a return type, since this is part of the RPC boundary. If you run the tests they should fail because of this.